src/main.rs

```diff
@@ -397,14 +406,17 @@ fn encode_token(matches: &ArgMatches) -> JWTResult<String> {
                 .collect(),
             _ => panic!("Invalid JSON provided!"),
         });
-    let expires = PayloadItem::from_string_with_name(matches.value_of("expires"), "exp");
+    let now = Utc::now().timestamp();
+    let expires = PayloadItem::from_timestamp_with_name(matches.value_of("expires"), "exp", now);
+    let not_before = PayloadItem::from_timestamp_with_name(matches.value_of("not_before"), "nbf", now);
+    let issued_at = PayloadItem::from_string_with_name(Some(&now.to_string()), "iat");
```

Comment:
Ok, last one: let's treat all duration/timestamp items the same, which means using `from_timestamp_with_name` for `iat` as well

```suggestion
    let issued_at = PayloadItem::from_timestamp_with_name(Some(&now.to_string()), "iat", now);
```

Reply:
I used the "Commit suggestion" feature, hope it works 😬